packages/calling/src/CallingClient/CallingClient.ts

```diff
@@ -491,6 +413,40 @@ export class CallingClient extends Eventing<CallingClientEventTypes> implements
   public getLines(): Record<string, ILine> {
     return this.lineDict;
   }
+
+  /**
+   * Retrieves call objects for all the active calls present in the client
+   */
+  public getActiveCalls(): Record<string, ICall[]> {
+    const activeCalls = {};
+    const calls = this.callManager.getActiveCalls();
+    Object.keys(this.lineDict).forEach((deviceId) => {
+      const {lineId} = this.lineDict[deviceId];
+      activeCalls[lineId] = [];
+      Object.keys(calls).forEach((correlationId) => {
+        if (calls[correlationId].lineId === lineId) {
+          activeCalls[lineId].push(calls[correlationId]);
+        }
+      });
+    });
+
+    return activeCalls;
+  }
+
+  /**
+   * Retrieves call objects for the connected call in the client
```

Comment:
Single call object is returned so please correct the same in decription

Reply:
Changed